language-design.md

```diff
@@ -38,7 +113,31 @@ How do the themes of _Growing a Language_ relate to the "sound lab" we did this
 
 **Response**
 
-
+In the sound lab, we were growing the sound library from the completed CS5 lab. We
+were improving the usability and fluency of that library. One thing we could have
+done to improve the sound library would be to overload the addition operator
+such that two sounds could be added to each other. In Growing a Language,
+Steele mentions "right now operators in the Java programming language can not
+be overloaded by the programmer, though names of methods may be overloaded. I 
+would like to change that." [Steele 1998]. That's an improvement to Java
+that would allow a similar improvement in a sound library. This runs along
+the same theme as a language's capability for growth. 
+
+The sound library we were using has the characteristics of a "small" language.
+It has basic capabilities of functions but everything writes to out.wav and you
+must retype the source file name every time you want to modify it. This
+provides many opportunities for the same kind of growth by wart removal 
+described in Steele: "Users will not put of with the warts for all time". 
+[Steele, 2012] We designed a way for users to no longer deal with warts. 
```

Comment:
I would add that these are something we considered "warts", but others might not. If we were doing this properly, we would have interviewed people interested in using the software for actually producing music.